tests/compiled_templates/mesh/communities/bird.cfg

```diff
@@ -2,8 +2,6 @@ function apply_communities ()
 {
       if ( net ~ 192.168.0.0/16 ) then {
           bgp_community.add((100, 6532));
-          bgp_community.add((5663, 12));
```

Comment:
Test uses basic diff to compare expected and actual bird.cfg, as we are using map to store communities, order is not guaranteed and diff would fail so, removing all except one community.